src/metabase/driver/sync.clj

```diff
@@ -7,16 +7,18 @@
   "Converts a schema pattern, as entered in the UI, into regex pattern suitable to be passed into [[re-pattern]].  The
   conversion that happens is from commas into pipes (disjunction), and wildcard characters (`*`) into greedy wildcard
   matchers (`.*`).  These only occur if those characters are not preceded by a backslash, which serves as an escape
-  character for purposes of this conversion.
+  character for purposes of this conversion.  Any whitespace before and after commas is trimmed.
 
   Examples:
     a,b => a|b
     test* => test.*
     foo*,*bar => foo.*|.*bar
+    foo  ,  ba*r  , baz => foo|ba.*r|baz
     crazy\\*schema => crazy\\*schema"
-  ^Pattern [schema-pattern]
-  (re-pattern (-> (str/replace schema-pattern #"(^|[^\\\\])\*" "$1.*")
-                  (str/replace #"(^|[^\\\\])," "$1|"))))
+  ^Pattern [^String schema-pattern]
+  (re-pattern (->> (str/split schema-pattern #",")
+                   (mapv (comp #(str/replace % #"(^|[^\\\\])\*" "$1.*") str/trim))
```

Comment:
just curious, is there a reason to use `mapv` here instead of just `map`?

Reply:
Hmm, I guess in this case it's a bit pointless (since we immediately just join it).  Sometimes I do it to make debugging easier, but that's overkill here.  The output of the `str/split` being fed in is already a vector, so I'm not sure if there is any micro level performance difference between `map` and `mapv` there.

Reply:
Changed it to `map` since I can't see any actual reason for `mapv`.